Community/bulk/bulk.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019 BlueCat Networks (USA) Inc. and its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from bluecat_portal.customizations.tools import tools
```

Comment:
This dependency in not included with Gateway. Makes this workflow difficult for people to download and use since they might not have the tools module.